python/testing/test_camera.py

```diff
@@ -86,6 +86,21 @@ def test_moves():
     camera.pitch(angle)
 
 
+def test_pan():
```

Comment:
nice, but needs a C++ test :)

Reply:
for some reason I thought python testing counted towards coverage

Reply:
The logic is that python is an optional dependency, so python tests are testing the bindings themselves, not the feature, which must be tested using c++